packages/common/src/templates/configuration/expo/index.ts

```diff
@@ -0,0 +1,13 @@
+import { ConfigurationFile } from '../types';
+
+const config: ConfigurationFile = {
+  title: 'app.json',
+  type: 'expo',
+  description: 'Configuration for your universal app.',
+  moreInfoUrl: 'https://docs.expo.io/versions/latest/workflow/configuration/',
+  partialSupportDisclaimer: `Native app development is not supported in Code Sandbox yet. You can download your project and run it locally, or copy your code to https://snack.expo.io to run your project in the Expo client.`,
```

Comment:
I'm thinking about decoupling Metro from React Native to run with Webpack, it would then be pretty straightforward to run Expo Client with Code Sandbox. Until then I could investigate snack-sdk integration, but that also could be kinda hard. 😅